### DISCUSSION TITLE
ufw firewall rules

### DISCUSSION THREAD

**Discussion by ghost** [DistroAV/DistroAV#893] [Q&A - Support]
### Running Ubuntu 22.04 (OBS 29.1.2) 
This machine has **strict firewall rules**

### Sending output to Debian 11 stable (OBS 26.1.2)
This machine **allows all** from the Ubuntu machine.

---

Everything works as long as I disable firewall or set it to allow all from the Debian machine. (I do not want this)
I just want to allow the machine in through what ports OBS is using. 

When ufw is enabled, I can't see the stream on the Debian machine. 
Ive tried allowing the following ports:
```
1935    
443                                
80                      
554                          
3389                                
3350                    
25                         
1716                              
5960                            
5961                             
631                            
9090                             
5001 
```

After allowing all of those through from the Debian machine I still couldn't view the stream .
Disabling firewall in Ubuntu immediately fixes the problem. 

So does anyone know how to set firewall rules so I don't have to disable or allow all traffic?
And why do I need to allow the Debian machine into the Ubuntu system in order to send an output to that OBS instance? 

Thank you :)

**Comment by Trouffman** [comment]
Please use the latest version and follow the install /pre-requisites steps.
